tkltest/generate/ui/generate_with_labels_analysis.py

```diff
@@ -108,23 +116,29 @@ def generate_selenium_api_tests(config, crawl_dir):
     empty_form_field_labels = 0
     eventable_dom_label_table = []
 
+    form_field_dom_label_table = []
+
+    heuristic_label.get_element_and_method_labels(crawl_paths)
+
     for path_num, crawl_path in enumerate(crawl_paths):
         # for each path create a jinja context for the test method to be generated
-        method_name = __create_method_name_for_path(crawl_path)
-        logging.info('Path {}: length={}, {}'.format(path_num, len(crawl_path), method_name))
-        if method_name in method_name_count:
-            method_name_count[method_name] = method_name_count[method_name] + 1
-            method_name = '{}_dup{}'.format(method_name, method_name_count[method_name])
-            logging.info('Creating duplicate test method: {}'.format(method_name))
+        method_eventables_path = __create_method_name_for_path(crawl_path)
+        logging.info('Path {}: length={}, {}'.format(path_num, len(crawl_path), method_eventables_path))
+        if method_eventables_path in method_eventables_path_count:
+            method_eventables_path_count[method_eventables_path] = method_eventables_path_count[method_eventables_path] + 1
+            method_eventables_path = '{}_dup{}'.format(method_eventables_path, method_eventables_path_count[method_eventables_path])
```

Comment:
@divtosz should we indeed create a duplicate method, or skip the duplicate path?

Reply:
@divtosz Why is the code being copied? Shouldn't the loop over tests in CrawlPaths.json be implemented once?

Reply:
No I wrote this file to run the test generation with a static file, without crawljax creating the paths, just for analysis purposes and to make testing faster